.github/workflows/plugin_upstream_testing_base.yml

```diff
@@ -41,14 +42,77 @@ jobs:
           builder: "${{ steps.buildx.outputs.name }}"
           context: "./"
           push: false
-          load: true
           tags: "${{ inputs.plugin_name }}/nautobot:${{ matrix.nautobot-version }}-py${{ env.PYTHON_VERSION }}"
           file: "./development/Dockerfile"
           cache-from: "type=gha,scope=nautobot-${{ matrix.nautobot-version }}"
           build-args: |
             NAUTOBOT_VER=${{ matrix.nautobot-version }}
             PYTHON_VER=${{ env.PYTHON_VERSION }}
+          outputs: "type=docker,dest=/tmp/nautobot_${{ matrix.nautobot-version }}.tar"
+      - name: Upload docker image artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: "docker_nautobot_${{ matrix.nautobot-version }}"
+          path: "/tmp/nautobot_${{ matrix.nautobot-version }}.tar"
+  unittest:
+    name: "Run Unit Tests"
+    needs: "build"
+    strategy:
+      fail-fast: false
+      matrix:
+        nautobot-version: ["develop", "ltm-1.6"]
+    runs-on: "ubuntu-20.04"
```

Comment:
Can we use 24.04 at this point, at least for new tasks?

Reply:
It will probably work fine for the CI base image

Reply:
Testing 24.04 [here](https://github.com/gsnider2195/nautobot-app-cookiecutter-test/actions/runs/10273784541)